scripts/bootstrap.sh

```diff
@@ -27,5 +27,4 @@ done
 pipenv install --dev --deploy "${pipenv_install_args[@]}"
 pipenv run nodeenv -p --node=16.14.0
 pipenv run npm install -g yarn@1.22.19
-pipenv run yarn global add @vue/cli@v4.5.15
```

Comment:
This command appears in a couple of workflows in `.github/workflows/`. Do we want to drop it from there too?

Also, `scripts/build_template_overrides.py` seems to call `vue-cli-service`. Should it be modified to call it through `yarn`, or are we relying on the fact that `build_template_overrides.py` itself is run by yarn, so should have the right environment to find `vue-cli-service` anyway?

Reply:
I'm not hitting the CI errors locally, so this was a blind attempt to try to fix it. That said, I would like to clean this up appropriately so that CI runs the same setup developers have. Good catch on calling `vue-cli-service` from inside that script. Although it's probably being run from yarn 99% of the time, I think I'll make sure it's called via yarn.

Reply:
>  That said, I would like to clean this up appropriately

Yeah, it’s always nagged me that `setup-bootstrap.sh` exists but isn’t used by the CI workflows. Either it should exist and be used everywhere, or not exist. If that’s part of what you’re referring to? If not, I’ve just had a good rant anyway.

Reply:
Exactly. I think it arrived there organically, but yeah, it bothered me, too.